sections/errorhandling/centralizedhandling.md

```diff
@@ -62,11 +67,16 @@ catch (error) {
 
 // Error handling middleware, we delegate the handling to the centralized error handler
 app.use(async (err: Error, req: Request, res: Response, next: NextFunction) => {
-  const isOperationalError = await errorHandler.handleError(err);
-  if (!isOperationalError) {
-    next(err);
-  }
+await errorHandler.handleError(err, res);
 });
+
+process.on("uncaughtException", error:Error => {
```

Comment:
In typescript we have to add parenthesis on single parameter arraow functions if we add type information: `(error:Error) => `